app/javascript/article-form/components/ArticleCoverImage.jsx

```diff
@@ -27,8 +27,7 @@ export class ArticleCoverImage extends Component {
     this.clearUploadError();
 
     if (validateFileInputs()) {
-      const { files: image } =
-        event instanceof DragEvent ? event.dataTransfer : event.target;
```

Comment:
Safari does not have a `DragEvent`. It is a `MouseEvent`. So switching to property check instead.